dbConfig.yml

```diff
@@ -1,8 +1,8 @@
 db:
   host: 34.228.205.4
```

Comment:
Can we have the IP address in Environment Variable as well?

Reply:
Any particular reason why?
This is not sensitive information per se.

Reply:
Maybe in regards to our moviecat. But I think the first step to avoid attack from enemies is to make your location secret. If they know your house, they can try to climb the wall. If they don't know your house, they have nowhere to start from... or perharps have a lot more work to do. I think it is sensitive. I always keep everything related to the database out of easy reach.

Reply:
Okay. Will do.